python/ball_launcher_beepy/ball_launcher_gui.py

```diff
@@ -1,333 +1,385 @@
-# ---------------------------------------------------------------

```

Comment:
General comment on this file:
There seems to be a lot of technical code (setting servo positions and the like) mixed with the GUI code.  If possible, it would be nice to separate this into separate modules, so one provides functions to control the hardware (independent of the GUI) and the GUI is just calling those functions.
I see that this might be a significant effort, though, so maybe not worth it as long as it works as is.  In any case it would probably be too much for this PR.